src/pages/CreateBurnerWallet.tsx

```diff
@@ -0,0 +1,143 @@
+import { AccentText, HeaderText, PostText } from 'components/Text'
+import { Sizes } from 'models/CharInCircle'
+import { useState } from 'preact/hooks'
+import Button from 'components/ui/Button'
+import CharInCircle from 'components/ui/CharInCircle'
+import TextInput from 'components/ui/TextInput'
+import Tooltip from 'components/ui/Tooltip'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  gap,
+  justifyContent,
+  textAlign,
+  width,
+} from 'classnames/tailwind'
+
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-8')
+)
+const topPart = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-4')
+)
+const bottomPart = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  justifyContent('justify-start'),
+  gap('gap-y-6')
+)
+
+const hintWrapper = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  gap('gap-x-1.5')
+)
+const buttonWithStatus = classnames(
+  display('flex'),
+  flexDirection('flex-col', 'sm:flex-row'),
+  alignItems('items-center'),
+  textAlign('text-center', 'sm:text-left'),
+  gap('gap-y-4', 'sm:gap-x-4')
+)
+const buttonClass = classnames(display('block'), width('w-full', 'sm:w-64'))
+
+export default function () {
+  const [state, setState] = useState(0)
```

Comment:
This is more like `stage`/`scene` if you want to use the number. Btw, numbers are not good for understanding, maybe you can make an `Enum`:
```TS
enum Scenes {
 0 = 'Landing',
 1 = 'Error',
 2 = 'CastPost'
}
Or something like this, please :)

Reply:
Changed it. But it is only for test and demonstration of UI. But anyway changed it